### DISCUSSION TITLE
VolumeControl unavailable after new install

### DISCUSSION THREAD

**Discussion by Guddi11** [MiczFlor/RPi-Jukebox-RFID#2054] [Q&A] [answered]
Hi, 
I had to newly setup my phoniebox recently. I copied the RPi-Jukebox Folder from the original SD card and reused this installation but otherwise tried to get everything setup from scratch.
The box itself does work again (RFID reader, audio out, part of the buttons, access via webapp) BUT

the volume control seems to have a problem. I cannot change the volume anymore, neither via webapp nor via buttons . 

`tail /var/log/syslog` shows the following problem:

```bash
/home/pi/RPi-Jukebox-RFID/scripts/playout_controls.sh: line 471: [: -ge: unary operator expected
```

Anybody any idea what that caused and how to fix it? I don't really dare fiddeling with the scripts, even though I understand what is happening.

**Comment by AlvinSchiller** [comment, marked as answer]
Hi,
which version did you update from? And to version 2.4 or develop?

Can you activate the "playout_controls" debug output and post it?
And also post your "mpd.conf" and "settings/global.conf" file?

**Reply by Guddi11** [reply]
I am on 2.4.0 (master) now. Where I come from I can’t be sure, as the SD card is not booting anymore. But I installed it end 2021. and it was also a release version.
I’ll check the rest of the questions later as I’m not home just now.

**Reply by Guddi11** [reply]
Ok, that's fun. After a fresh boot this morning the problem is gone. I can change volume via buttons and webapp alike. Strange it didn't already get fixed when I rebooted yesterday (twice...).
Anyway, I learned how to use the debug mode of the Phoniebox, which is great. Thanks @AlvinSchiller for that and for your help! :)
